Formula/envoy.rb

```diff
@@ -0,0 +1,45 @@
+class Envoy < Formula
+  desc "Cloud-native high-performance edge/middle/service proxy"
+  homepage "https://www.envoyproxy.io"
+  url "https://github.com/envoyproxy/envoy.git",
+      tag:      "v1.15.2",
+      revision: "c2a3cbd3634c490e613dfddc05483b74e5da29aa"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "bazelisk" => :build
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
+  depends_on "go" => :build
+  depends_on "libtool" => :build
+  depends_on "ninja" => :build
+
+  def install
+    args = %w[
+      --curses=no
+      --show_task_finish
+      --verbose_failures
+      --action_env=PATH=/usr/local/bin:/opt/local/bin:/usr/bin:/bin
+      --test_output=all
+    ]
+    system "bazelisk", "build", *args, "//source/exe:envoy-static"
```

Comment:
Please use `Formula["bazelisk"].opt_bin"/bazelisk"` here to ensure we point to the correct formula.